sidebars.js

```diff
@@ -113,7 +113,15 @@ module.exports = {
           type: "category",
           label: "Technical Guide",
           collapsed: true,
+          
+          insight-page-update
+          items: [
+            "contributing/technical/setup-repo-with-git",
+            "contributing/technical/resolve-merge-conflicts",
+          ],
+      
```

Comment:
I'm not sure what's going on here.
You changes doesn't require updating the `sidebars.js` and the changes here is causing error and terminated the docs locally.
Can you please update your branch before working on your changes?